packages/commercetools/theme/components/InstagramFeed.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
How about putting it in default layout and then just specifying different layout for checkout? In that way we could avoid repetition)